.changeset/five-carpets-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Fixes the case where Connection queries with @limit in the type would generate unnecessary cypher with extra node/edge projection.
```

Comment:
```suggestion
Fixes the case where Connection queries with `@limit` in the type would generate unnecessary cypher with extra node/edge projection.
```

We want to avoid pinging this user :smile: